src/components/explore/DistributionChart.svelte

```diff
@@ -0,0 +1,66 @@
+<script>
+  import { scaleLinear } from 'd3-scale';
+  import { tooltip as tooltipAction } from '@graph-paper/core/actions';
+  import { distributionComparisonGraph } from '../../utils/constants';
+
+  export let density;
+  export let topTick;
+  export let sampleCount;
+  export let tooltipLocation;
+
+  let height = distributionComparisonGraph.height/2;
+  let color = 'var(--digital-blue-350)';
+  let binSelector = 'bin';
+  let offsetX = distributionComparisonGraph.left + 7;
+  let width = distributionComparisonGraph.width - distributionComparisonGraph.right - distributionComparisonGraph.left
+  let maxHeight = height - distributionComparisonGraph.top
+  let minHeight = distributionComparisonGraph.bottom
+  let formatPercent = (t) => Intl.NumberFormat('en-US', {style: 'percent', maximumFractionDigits: 2}).format(t)
+  let formatCompact = (t) => Intl.NumberFormat('en', { notation: 'compact' }).format(t)
+
+  $: y = scaleLinear()
+    .domain([0, topTick])
+    .range([minHeight, maxHeight]);
+
+  const bucketWidth = width*1.0101/density.length
+  const spaceBetweenBars = bucketWidth/10
+  const barOffsetX = spaceBetweenBars/2
+  const barWidth = bucketWidth - spaceBetweenBars
+</script>
+<style>
+  .hovd {
+    opacity: 0;
+  }
+  .hovd:hover {
+    opacity: 0.3;
+    z-index: 999;
+  }
```

Comment:
Let's add some padding between the y-axis labels, they look too close together.

<img width="1500" alt="CleanShot 2023-09-14 at 13 50 45@2x" src="https://github.com/mozilla/glam/assets/28797553/1c9636d8-7873-46a9-9b14-5bb5fef69c05">

Reply:
The labels are drawn by graph-paper and it looks like they don't have enough space to be displayed side-by-side because of the amount of buckets of that histogram. One solution is to skip some of them, but I don't think graph-paper supports that